packages/workspace/src/tasks-runner/run-command.spec.ts

```diff
@@ -1,7 +1,300 @@
 import { TasksRunner } from './tasks-runner';
 import defaultTaskRunner from './default-tasks-runner';
-import { getRunner } from './run-command';
+import { createTasks, getRunner } from './run-command';
 import { NxJson } from '../core/shared-interfaces';
+import { DependencyType, ProjectGraph } from '@nrwl/devkit';
+
+describe('createTasks', () => {
```

Comment:
I think you should add the following test case:

- what if you add a dep on self but the target isn't there?